### ISSUE TITLE
STOP invoked while a button is pressed

### ISSUE THREAD

**Issue by avivace** [gbdev/pandocs#110]
Reducing_Power_Consumption.md:

> If STOP is invoked while a button is held down, it might enter a bugged
STOP mode. Which functions like a HALT instead of STOP, so requires
interrupts to be enabled.

Is this true? Can we work on a better phrasing?

**Comment by CasualPokePlayer** [comment]
This is true. I have verified myself using my GB Player this is how bugged stop acts. This is also why I suspect the part of the joypad interrupt in the old description was thought to be needed (it's not). However, I don't think we can say it is literally halt mode. I might be misremembering this, but I recall rDIV not ticking in bugged stop mode (timer interrupt never came), and I have not verified whether or not the halt bug is present with this weird mode. To note, this also affects speed switching, and I am not sure of the behavior of exiting speed switch via interrupt in bugged stop (does speed switch still happen? Are there other consequences for doing this?).
